cmd/kubeadm/app/phases/kubelet/config_test.go

```diff
@@ -109,6 +112,72 @@ func TestApplyKubeletConfigPatches(t *testing.T) {
 	}
 }
 
+func TestApplyKubeletConfigPatchFromFile(t *testing.T) {
+	tests := []struct {
+		name           string
+		kubeletConfig  []byte
+		patchContent   []byte
+		expectError    bool
+		expectedResult []byte
+	}{
+		{
+			name:           "apply new field",
+			kubeletConfig:  []byte("apiVersion: kubelet.config.k8s.io/v1beta1\nkind: KubeletConfiguration\n"),
```

Comment:
let's create a constant for `"apiVersion: kubelet.config.k8s.io/v1beta1\nkind: KubeletConfiguration\n"`
you can put it above `tests := ...`